Documentation/ceph-cluster-crd.md

```diff
@@ -317,14 +324,31 @@ If you want to learn more, please read
 
 Based on the configuration, the operator will do the following:
 
-  1. if only the `public` selector is specified both communication and replication will happen on that network
-  2. if both `public` and `cluster` selectors are specified the first one will run the communication network and the second the replication network
+  1. If only the `public` selector is specified, all communication will happen on that network
+```yaml
+  network:
+    provider: multus
+    selectors:
+      public: rook-ceph/rook-public-nw
+```
+  2. If only the `cluster` selector is specified, the internal cluster traffic* will happen on that network. All other traffic to mons, OSDs, and other daemons will be on the default network.
+* Internal cluster traffic includes OSD heartbeats, data replication, and data recovery
```

Comment:
I'd suggest moving this * below the example of bullet 3. It is also rendering as a bullet instead of a note, so perhaps it needs to be escaped to remain as a *

Reply:
done